docs/Documentation/Applications/starccm.md

```diff
@@ -0,0 +1,76 @@
+# Running STAR-CCM+ Software
+
+*Learn how to run STAR-CCM+ software on the the Eagle system.*
+
+For information about the software's features, see the [STAR-CCM+
+website](https://mdx.plm.automation.siemens.com/star-ccm-plus).
+
+STAR-CCM+ can be run interactively on Eagle using X windows. The network
+licenses are checked out from the license server running on wind-lic.nrel.gov.
+
+First, build your simulation `<your_simulation.sim>` on your workstation and
+copy to your `/scratch/$USER/<sim_dir>` directory on Eagle:
+
+```
+ls /scratch/$USER/sim_dir
+your_simulation.sim
+```
+
+Create a Slurm script `<your_scriptfile>` as shown below:
+
+``` bash
```

Comment:
It might improve readability to put the example script in a drop down box (I'm looking at NERSC application pages for examples: https://docs.nersc.gov/applications/vasp/#cori-haswell). It's a short script though so I think either way is fine.

Reply:
✔️